### ISSUE TITLE
Reports for pretrained bias model (positive control)

### ISSUE THREAD

**Issue by Rafaelsoler13** [kundajelab/chrombpnet#101]
Hello!

First thanks for this fantastic tool :)

I wanted to ask about the possibility of including the output.pdf or html file for the pretrained bias model, just to check what the output should look like to be a good quality bias model, and also to check if we can replicate the bias model pre-trained with the tool installed in our machines (positive control), corroborating that everything works well (in this case a seed would be necessary).

Best regards,

Rafael

**Comment by panushri25** [comment]
Hello Rafael,

Thank you for your encouraging words. 

The bias model shared here (https://github.com/kundajelab/chrombpnet/wiki/ChromBPNet-training) is trained using the arguments here (https://github.com/kundajelab/chrombpnet/wiki/Bias-model-training). So the reports for the bias model are already provided with the repo here- https://github.com/kundajelab/chrombpnet/wiki/Bias-model-training. Are you able to find it? 


The seed is set by default (look at optional input arguments) and the model provided in the repo is trained with the default seed and also the exact command line argument posted so you can use it to verify reproducibility.  Also do note that training on the same GPU twice with the exact same code, ensures similar results but not exact reproducibility in terms of model weights learnt etc (because of the dynamic job assignment on the GPU kernels I believe.) 

Hope this answers your question. 

Best,
Anu

**Comment by Rafaelsoler13** [comment]
Yes! I didn't see it before, sorry. It's this report, isn't it? https://storage.googleapis.com/chrompnet_data/bias_model/eEvaluation/k562_overall_report.html

Ok, even though the results may not be 100% reproducible, I guess it won't vary the result much to check that everything is working fine.

Thanks for the quick reply :)

**Comment by panushri25** [comment]
The link you shared has typo I think - but looks like the correct one

https://storage.googleapis.com/chrombpnet_data/bias_model/evaluation/k562_overall_report.html
